src/OgAccess.php

```diff
@@ -184,16 +206,16 @@ public function userAccess(EntityInterface $group, $operation, AccountInterface
   public function userAccessEntity($operation, EntityInterface $entity, AccountInterface $user = NULL) {
     $result = AccessResult::neutral();
 
-    // Entity isn't saved yet.
-    if ($entity->isNew()) {
-      return $result->addCacheableDependency($entity);
-    }
-
     $entity_type = $entity->getEntityType();
     $entity_type_id = $entity_type->id();
     $bundle = $entity->bundle();
 
-    if (Og::isGroup($entity_type_id, $bundle)) {
+    if ($this->groupManager->isGroup($entity_type_id, $bundle)) {
+      // Entity isn't saved yet.
+      if ($entity->isNew()) {
+        return $result->addCacheableDependency($entity);
+      }
```

Comment:
This is moved in here to satisfy the tests. There is a test specifically added to test that `AccessResult::neutral()` is returned when a new group is passed in. The reason for this is not entirely clear to me, it's not documented here nor in the test, but since there's a test for it I'm assuming that this is important for some reason. I have not investigated any further since it's out of scope for this PR.